src/location-distance-utils.c

```diff
@@ -21,31 +21,33 @@
 
 #include "location-distance-utils.h"
 
-static double d2r(double deg)
-{
-	return (deg * M_PI / 180);
-}
+#ifndef M_PI
+#define M_PI 3.14159265358979323846
+#endif
+
+// scaling factor from degrees to meters at equator
+// == DEG_TO_RAD * radius of earth
+static float LOCATION_SCALING_FACTOR = 111318.84502145034f;
```

Comment:
why variable and not "#define LOCATION_SCALING_FACTOR 111318.84502145034"

Reply:
I'm sorry for replying late, I was very busy, and yes it could be #define but it's a good practice to prefer const over #define